tqdb/dbr.py

```diff
@@ -40,44 +44,50 @@ def read(dbr):
 
     """
     try:
-        dbr_file = open(dbr)
-    except FileNotFoundError:
-        logging.debug(f'No file found for {dbr}')
-        return {}
-    except PermissionError:
-        logging.debug(f'Tried opening a directory {dbr}')
-        return {}
+        with open(dbr) as dbr_file:
+            result = {}
```

Comment:
What's the benefit of not try/catching just the file opening? The rest of the code has no exceptions to try/catch here, and with this setup the entire block is indented needlessly?

Reply:
I added the `with` so that the file would be automatically closed by Python at the end of that block. Therefore it's necessary to put the catches on the `with` block. I suppose the only other thing you could catch is `IOError`, since that could occur at any time during reading the file.

I could replace the `with` with a close in a finally block, but I think `with` is better. What I can definitely do though is move everything after creating `properties` outside of the block. Since the dict creation cannot be done lazily, we must have read the whole file by the time the dict is created, so we can close it at that point.